src/backend/distributed/sql/citus--11.0-2--11.1-1.sql

```diff
@@ -6,3 +6,14 @@ DROP FUNCTION pg_catalog.worker_hash_partition_table(bigint, integer, text, text
 DROP FUNCTION pg_catalog.worker_merge_files_into_table(bigint, integer, text[], text[]);
 DROP FUNCTION pg_catalog.worker_range_partition_table(bigint, integer, text, text, oid, anyarray);
 DROP FUNCTION pg_catalog.worker_repartition_cleanup(bigint);
+
+SET search_path = 'pg_catalog';
+
+CREATE OR REPLACE FUNCTION lock_relation_if_exists(table_name text, lock_mode text, nowait boolean)
```

Comment:
we typically change the UDFs under ` src/backend/distributed/sql/udfs` folder. `lock_relation_if_exists` is an old UDF that does not have a folder, but we can create now and then include here like `#include "udfs/lock_relation_if_exists/11.0-1.sql"`

See `src/backend/distributed/sql/citus--10.2-4--11.0-1.sql ` for such examples.